src/index.html

```diff
@@ -164,18 +164,40 @@ <h6 class="block__sub block__flush-top">Key tools</h6>
         <aside class="content_sidebar">
             <!-- STAY TUNED -->
           <section class="block u-mt0">
-          <header class="tabbed-header tabbed-pad-top">
-            <h3 class="subhead tabbed-heading">Stay tuned</h3>
-          </header>
-          <p>Sign up for our email list, and we’ll let you know when we add more tools and resources.</p>
-
-          <form class="signup2 inline-signup" id="signup" action="https://public.govdelivery.com/service/process_ss.xml" method="GET" data-thanks="Thanks, we’ll be in touch!">
-            <input type="hidden" name="code" value="USCFPB_46">
-            <label for="email" class="u-visually-hidden">Email address</label>
-            <input type="email" id="email" name="email" placeholder="Enter your email address">
-            <fieldset class="submit_btn"><p><button id="beta-btn" class="btn">Sign up</button></p></fieldset>
-          </form>
-        </section>
+            <div class="o-email-signup">
+              <h2 class="header-slug">
+                <span class="header-slug_inner">
+                  Stay tuned
+                </span>
+              </h2>
+
+              <form id="{{ ('o-email-signup_' ~ range(1, 100) | random) }}"
```

Comment:
Does this id need to be unique on the page? If so, this needs another approach as it will potentially collide with other forms that use this id scheme.